File: Python Exams/Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/03. Easter Trip/03. Easter Trip.py
destination = input()
data = input()
nights = int(input())

price_dict = {
   'France' : {'21-23' : 30, '24-27' : 35, '28-31' : 40 },
   'Italy' : {'21-23' : 28, '24-27' : 32, '28-31' : 39 },
   'Germany' : {'21-23' : 32, '24-27' : 37, '28-31' : 43 },
}

print(f'Easter trip to {destination} : {price_dict[destination][data] * nights:.2f} leva.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/03. Easter Trip/03. Easter Trip.js
function easterTrip(destination, data, nights) {
   
   const priceObj = {
      'France' : {'21-23' : 30, '24-27' : 35, '28-31' : 40 },
      'Italy' : {'21-23' : 28, '24-27' : 32, '28-31' : 39 },
      'Germany' : {'21-23' : 32, '24-27' : 37, '28-31' : 43 },
   }

   return `Easter trip to ${destination} : ${(priceObj[destination][data] * nights).toFixed(2)} leva.`
}

//console.log(easterTrip('Germany', '24-27', 5))